migrate/README.md

```diff
@@ -0,0 +1,8 @@
+# Migration Framework
+To have reproducible environments, support rollbacks and make sure that every change is only executed once we use [Golang Migrate](https://github.com/golang-migrate/migrate/tree/master) as migration framework.
```

Comment:
Add that we also do the initial database setup using `migrate`, and that the migrations in `not_supported_yet` are not executed, but used as an example and a place to store work that was not finished yet.